src/components/LogoAndLinkButton/LogoAndLinkButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { Stack, useDeskproAppTheme } from "@deskpro/app-sdk";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowUpRightFromSquare } from "@fortawesome/free-solid-svg-icons";
+
+import { SurveyMonkeyLogo } from "../SurveyMonkeyLogo/SurveyMonkeyLogo";
+import { StyledLink } from "../../styles";
+
+export const LogoAndLinkButton = ({ endpoint }: { endpoint: string }) => {
```

Comment:
we already have such a component in the [app-sdk](https://github.com/DeskproApps/app-sdk/blob/master/src/ui/components/ExternalIconLink/ExternalIconLink.tsx)

Reply:
I think we should remove this component if we don't use it